src/Modules/DatePicker.react.js

```diff
@@ -386,7 +386,11 @@ class DatePicker extends Component {
                 this.props.onChange(dateObj);
             }
         } else {
-            this.props.onChange(date.unix());
+            if (date) {
```

Comment:
is `if (moment.isMoment(date))` maybe slightly safer here?
https://momentjs.com/docs/#/query/is-a-moment/

Being a mostly C# dude I tend to be leary of simple truthiness or falsiness checks and lean towards more precise checks (using the various helpers in lodash and such, which I _love_).